src/test/java/ScattTest.java

```diff
@@ -9,13 +13,68 @@
  * @author Erik Cole
  */
 public class ScattTest {
+    /**
+     * Get a string representing a path to a project test resource.
+     * Project test resources should be put in src/test/resources.
+     * @param resName The name of the resource file
+     * @return String representation of path to resource
+     */
+    private String getTestResource(String resName) {
```

Comment:
I think that maybe we should rename it to `getTestResourcePath` though. The resource doesn't even need to exist for us to use this, and it's not like it's returning a File object or the binary contents of the resource.